src/pbt/entities/project.py

```diff
@@ -368,4 +368,4 @@ def load_pipeline_folder(self, pipeline_id: str):
             path = ".prophecy/{}".format(pid)
             return Project(path, pid).load_pipeline_folder(path)
         else:
-            return None
```

Comment:
This can back-fire in some case where the check is for non-None.

Reply:
Changed all usages from None to empty. Although empty was handled